easybuild/easyconfigs/c/CP2K/fix_compile_date_lastcvs.patch

```diff
@@ -1,14 +1,11 @@
---- makefiles/Makefile.orig	2010-12-10 05:35:15.000000000 +0100
-+++ makefiles/Makefile	2011-01-24 10:14:35.000000000 +0100
-@@ -149,9 +149,9 @@
+--- makefiles/Makefile.orig	2012-08-21 14:35:17.000000000 +0300
++++ makefiles/Makefile	2013-11-12 12:41:29.361582641 +0200
+@@ -194,7 +194,7 @@
```

Comment:
@boegel: Please ignore this. It is my mistake...if you notice in my config files I am using 'CP2K-2.4.0-fix_compile_date_lastsvn.patch' which was already in EasyBuild. I will correct my PR so as to delete this patch file from it.